# HG changeset patch
# User Albert Cervera i Areny <albert.nan.tic@gmail.com>
Close template1 connection on database restore.

Issue4717
review6241002

Index: trytond/trytond/backend/postgresql/database.py
===================================================================

--- a/trytond/trytond/backend/postgresql/database.py
+++ b/trytond/trytond/backend/postgresql/database.py
@@ -151,6 +151,7 @@
         database.create(cursor, database_name)
         cursor.commit()
         cursor.close()
+        database.close()

         cmd = ['pg_restore', '--no-owner']
         env = {}

